marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployHubAmpsCommand.java

```diff
@@ -119,32 +63,13 @@ public void undo(CommandContext context) {
         String serverVersion = versions.getMarkLogicVersion();
         logger.info("Choosing amp uninstall based on server version " + serverVersion);
 
-        if (serverVersion.startsWith("9.0-5")) {
-            logger.info("Using non-SSL-compatable method for 9.0-5 servers");
-            String modulesDatabaseName = hubConfig.getAppConfig().getModulesDatabaseName();
-            ManageConfig manageConfig = context.getManageClient().getManageConfig();
-            String securityUsername = manageConfig.getSecurityUsername();
-            String securityPassword = manageConfig.getSecurityPassword();
-            DatabaseClient installerClient = DatabaseClientFactory.newClient(
-                hubConfig.getHost(),
-                8000,
-                "Security",
-                new DatabaseClientFactory.DigestAuthContext(securityUsername, securityPassword)
-            );
-            //new AmpsInstaller(securityStagingClient).unInstallAmps(modulesDatabaseName);
-            ServerEvaluationCall call = installerClient.newServerEval();
-            try (InputStream is = new ClassPathResource("installer-util/uninstall-amps.xqy").getInputStream()) {
-                String ampCall = IOUtils.toString(is, "utf-8");
-                is.close();
-                ampCall = ampCall.replaceAll("data-hub-MODULES", modulesDatabaseName);
-                call.xquery(ampCall);
-                call.eval();
-            } catch (IOException e) {
-                throw new DataHubConfigurationException(e);
-            }
+        if (serverVersion.matches("^[9]\\.0-([6789]|[0-9]{2,})(\\.\\d+)?")) {
+            // only on 9.0-5 are amps uninstalled at all.
+            logger.warn("Amps from uninstalled data hub framework to remain, but are disabled.");
+            super.execute(context);
```

Comment:
We used to uninstall amps in 9.0-5 and not in any other version. Now it looks like we are uninstalling in all other versions except 9.0-5 ?

Reply:
Yeah, let's make sure that we're consistent in the uninstallation of amps.

Reply:
Now checking for "-SNAPSHOT" to determine if amps are uninstalled. https://github.com/marklogic/marklogic-data-hub/blob/b437b77f29ac6b73c18e963cd4db40a5fa5d70fc/marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployHubAmpsCommand.java#L67

Reply:
This should split the middle between wanting one path to handle amps, while also not slowing our tests to a crawl. @srinathgit good for you?